wureset.ps1

```diff
@@ -0,0 +1,1110 @@
+## AUTHOR:	Manuel Gil.
+## ==================================================================================
+## PowerShell Conversion by Neil Haagensen
+## Verson 0.0.1 2023-12-14 - DO NOT RUN! Still has unconverted DOS BAT code.
+## Started conversion 
+## Verson 0.0.2 2023-12-26 - DO NOT RUN! Still has unconverted DOS BAT code.
+## Refactored all anonymous functions into proper PS functions
+## Verson 0.0.3 2023-12-28 - DO NOT RUN! Still has unconverted DOS BAT code.
+## Solved sd.exe problem continued refactoring code.
+## Version 0.0.4 2023-12-28 - Reached EoF
+## Corrected error in self-elevation
+## Main Menu is now live.
+## Version 1.0 2023-12-29
+## Corrected lingering incorrect calls to Print function
+## Corrected typos in regedit function
+## Fixed commands for gpupdate, chkdsk, and restart functions
+## TODO: 
+## ===============================================================================
+
+
+## Set console.
+## void mode();
+## /************************************************************************************/
+
+#:mode
+#$SetMode = {
+function initialize {
+	#Set-PSDebug -Off
+	$host.ui.RawUI.WindowTitle = "Reset Windows Update Tool."
+    $Name = $null
+    $Family = $null
+    $currentDate = Get-Date -Format "yyyy-MM-dd"
+	#mode con cols=90 lines=36
+	#color 17
+	clear
+    GetValues
+	#goto getValues
+}
+
+## /************************************************************************************/
+
+
+## Print Top Text.
+##		@param - text = the text to print (%*).
+## void print(string text);
+## /*************************************************************************************/
+
+#:print
+function PrintTop {
+    param(
+        [string]$Title
+    )
+	clear
+	Write-Host "`n$Name [Version: $Version]"
+    Write-Host "Reset Windows Update Tool."
+    Write-Host "`n`n$Title"
+}
+
+## /*************************************************************************************/
+
+
+## Add Value in the Registry.
+##		@param - key = the key or entry to be added (%~1).
+##				value = the value to be added under the selected key (%~2).
+##				type = the type for the registry entry (%~3).
+##				data = the data for the new registry entry (%~4).
+## void addReg(string key, string value, string type, string data);
+## /*************************************************************************************/
+
+#:addReg
+function AddRegistryValue {
+    param (
+        [string]$Key,
+        [string]$Value,
+        [string]$Type,
+        [string]$Data
+    )
+	reg add "$Key" /v "$Value" /t "$Type" /d "$Data" /f
+}
+## /*************************************************************************************/
+
+
+## Load the system values.
+## void getValues();
+## /************************************************************************************/
+#:getValues
+function GetValues {
+#	for /f "tokens=4 delims=[] " %%a in ('ver') do set version=%%a
+    $Version = systeminfo | findstr /B /C:"OS Version"
```

Comment:
The output of [`systeminfo`](https://ss64.com/nt/systeminfo.html) is localized. The output of [`ver`](https://ss64.com/nt/ver.html) is not localized. Or have a look at some alternatives: https://ss64.com/ps/syntax-ver.html.